dietpi/dietpi-software

```diff
@@ -14536,6 +14619,15 @@ _EOF_
 
 		fi
 
+		software_id=186 # IPFS Node
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
+
+			Banner_Uninstalling
+			systemctl disable --now ipfs
+			rm -r /usr/local/bin/ipfs /mnt/dietpi_userdata/ipfs /etc/systemd/system/ipfs.service /etc/bashrc.d/ipfs.sh
```

Comment:
Check out other uninstalls where services are involved. To assure that really all related files are removed and avoid error messages when files have been removed before:
```
if [[ -f '/etc/systemd/system/ipfs.service' ]]
then
	systemctl disable --now ipfs
	rm -v /etc/systemd/system/ipfs.service
fi
[[ -d '/etc/systemd/system/ipfs.service.d' ]] && rm -R /etc/systemd/system/ipfs.service.d
rm -Rf /usr/local/bin/ipfs /mnt/dietpi_userdata/ipfs /etc/bashrc.d/ipfs.sh
```